lib/koala/api/graph_api.rb

```diff
@@ -500,7 +500,14 @@ def graph_call(path, args = {}, verb = "get", options = {}, &post_processing)
         options = {:appsecret_proof => true}.merge(options) if @app_secret
         result = api(path, args, verb, options) do |response|
           error = check_response(response.status, response.body)
-          raise error if error
+          if error
+            raise error
+          elsif response.status.to_i == 304
+            return {
+              "message" => "Etags sent with request match response",
```

Comment:
yeah I am not sure that Fb returns a 304 for anything else, but since we don't know and I am guessing this will change over time, I will make it more general